neqo-qpack/src/qpack_send_buf.rs

```diff
@@ -8,14 +8,14 @@ use std::ops::Deref;
 
 use neqo_common::Encoder;
 
-use crate::{huffman::encode_huffman, prefix::Prefix};
+use crate::{huffman, prefix::Prefix};
 
 #[derive(Default, Debug, PartialEq, Eq)]
-pub struct QpackData {
+pub struct Data {
```

Comment:
This name is awful.  What is it?  From the implementation, it looks like a buffer, not a unit of data.

Not for this PR, but this looks like it could be replaced by a trait that defines the encode_prefixed_encoded_int (also a name that could be better) or maybe just the encode_literal methods and attaches them to `neqo_common::Encoder`.